server/test/views/questiontypes/ApplicantQuestionRendererFactoryTest.java

```diff
@@ -42,4 +52,19 @@ public void rendererExistsForAllTypes(QuestionType type) throws UnsupportedQuest
     assertThat(renderedContent).contains("Sample question text");
     assertThat(renderedContent).doesNotContain("help text");
   }
+
+  @Test
+  @Parameters(method = "compositeTypes")
```

Comment:
Nit: When we add new question types it might be possible to forget to update here. Consider having the parameterization pass every question type in.

Then we can check if it's in the set of compositeTypes and assert that fieldset is in it. Otherwise, we'd assert that fieldset isn't in it.

Reply:
Sounds good, done!